systemd/ua-reboot-cmds.service

```
@@ -7,7 +7,6 @@
 
 [Unit]
 Description=Ubuntu Pro reboot cmds
-ConditionPathExists=/var/lib/ubuntu-advantage/marker-reboot-cmds-required
 ConditionPathExists=/var/lib/ubuntu-advantage/private/machine-token.json
```

Something like this might be helpful
```suggestion
ConditionPathExists=|/var/lib/ubuntu-advantage/marker-reboot-cmds-required
ConditionPathExists=|/var/lib/ubuntu-advantage/marker-only-series
ConditionPathExists=/var/lib/ubuntu-advantage/private/machine-token.json
```